dist/style.css

```diff
@@ -298,56 +312,151 @@ html.inboxy.dark-theme .date-row {
+}
+html.inboxy .date-row span:last-of-type {
     margin-right: 24px;
 }
 
-html.inboxy.dark-theme .date-row .archive-bundle {
-    background-image: url('chrome-extension://__MSG_@@extension_id__/assets/light-check-all.svg');
-    opacity: 0.7;
+html.inboxy .date-row .archive-bundle,
+html.inboxy .date-row .delete-bundle,
+html.inboxy .date-row .select-bundle {
+    background-size: 20px;
+    margin-right: 12px;
 }
 
-html.inboxy.dark-theme .date-row .archive-bundle:before {
-    background-color: rgb(255, 255, 255, 0.2);
+html.inboxy.dark-theme .date-row .archive-bundle,
+html.inboxy.dark-theme .date-row .delete-bundle,
+html.inboxy.dark-theme .date-row .select-bundle {
+    opacity: 0.7;
 }
 
-@-moz-document url-prefix() {
-    html.inboxy .date-row .archive-bundle {
-        background-image: url('moz-extension://__MSG_@@extension_id__/assets/check-all.svg');
-    }
-
-    html.inboxy.dark-theme .date-row .archive-bundle {
-        background-image: url('moz-extension://__MSG_@@extension_id__/assets/light-check-all.svg');
-    }
+html.inboxy.dark-theme .date-row .archive-bundle:before,
+html.inboxy.dark-theme .date-row .delete-bundle:before,
+html.inboxy.dark-theme .date-row .select-bundle:before {
+    background-color: rgb(255, 255, 255, 0.2);
 }
 
 /** 
  * Archive button 
  */
+html.inboxy .bundle-row .archive-bundle,
+html.inboxy .date-row .archive-bundle {
+    background-image: url('chrome-extension://__MSG_@@extension_id__/assets/check-all.svg');
+}
 html.inboxy .brq,
 html.inboxy .ar8 {
     background-image: url('chrome-extension://__MSG_@@extension_id__/assets/check.svg');
 }
 
+html.inboxy.dark-theme .bundle-row .archive-bundle,
+html.inboxy.dark-theme .date-row .archive-bundle {
+    background-image: url('chrome-extension://__MSG_@@extension_id__/assets/light-check-all.svg');
+}
 html.inboxy.messages-dark-theme .brq,
 html.inboxy.dark-theme .ar8 {
     background-image: url('chrome-extension://__MSG_@@extension_id__/assets/light-check.svg');
 }
 
 @-moz-document url-prefix() {
+    html.inboxy .bundle-row .archive-bundle,
+    html.inboxy .date-row .archive-bundle {
+        background-image: url('moz-extension://__MSG_@@extension_id__/assets/check-all.svg');
+    }
     html.inboxy .brq,
     html.inboxy .ar8 {
         background-image: url('moz-extension://__MSG_@@extension_id__/assets/check.svg');
     }
 
+    html.inboxy.dark-theme .bundle-row .archive-bundle,
+    html.inboxy.dark-theme .date-row .archive-bundle {
+        background-image: url('moz-extension://__MSG_@@extension_id__/assets/light-check-all.svg');
+    }
     html.inboxy.messages-dark-theme .brq,
     html.inboxy.dark-theme .ar8 {
         background-image: url('moz-extension://__MSG_@@extension_id__/assets/light-check.svg');
     }
 }
 
+html.inboxy.hide-archive-action .bundle-row .archive-bundle,
+html.inboxy.hide-archive-action .date-row .archive-bundle,
+html.inboxy.hide-archive-action .brq,
+html.inboxy.hide-archive-action .ar9 {
```

Comment:
This is going to hide some of Gmail's buttons; I think the new settings should only apply to bulk actions, to keep things simple.